main.py

```diff
@@ -69,12 +69,12 @@
     # Check if the message author has the Game Master role
     isGameMaster = False
     if message.guild == gamelog_channel.guild:
-        if game_master in [y.id for y in message.channel.guild.get_member(message.author.id).roles]:
+        if game_master in [y.id for y in message.author.roles]:
             isGameMaster = True
 
     isAdmin = False
     if message.guild == gamelog_channel.guild:
-        if administrator in [y.id for y in message.channel.guild.get_member(message.author.id).roles]:
+        if administrator in [y.id for y in message.author.roles]:
```

Comment:
this will not work in a `PrivateChannel`

Reply:
That's why the line above make sure that line will never be triggered in a DM.

Reply:
*makes

Reply:
I'm unsure if a `PrivateChannel` is a channel with permissions set, the API docs don't make it clear.